README.md

```diff
@@ -314,52 +315,95 @@ module.exports = {
 };
 ```
 
-## Configuration
+## Underreact configuration file
 
-**No configuration is necessary to get started.** On most production projects you'll want to set at least a few of the "Basic configuration" options. "Advanced configuration" options shouldn't be necessary in *most* cases ... but you never know.
+Underreact expects a `underreact.config.js` file to exist at the root of your project. Please note that **No configuration is necessary to get started.** On most production projects you'll want to set at least a few of the [configuration object](#configuration-object-properties) properties.
```

Comment:
A couple typos in here:

expects a > expect an
No configuration > no configuration